app/models/chef.rb

```diff
@@ -1,4 +1,6 @@
 class Chef < ApplicationRecord
-   validates_presence_of :name
-   has_many :dishes
-end
\ No newline at end of file
+  validates_presence_of :name
+  has_many :dishes
+  # has_many :dish_ingredients, through: :dishes
```

Comment:
You are correct here in that you don't need this commented out association - once the association is set up between dishes and ingredients, you can just use line 5 as if they had a direct line between them